pkg/preparer/orchestrate.go

```diff
@@ -174,21 +169,23 @@ func (p *Preparer) verifySignature(manifest pods.Manifest, logger logging.Logger
 		}
 		return false
 	}
-	return true
-}
-
-func (p *Preparer) installAndLaunchPod(newManifest *pods.Manifest, pod Pod, logger logging.Logger) bool {
-	// do not remove the logger argument, it's not the same as p.Logger
 
-	if newManifest.ID() != POD_ID && p.podIdForbidden(newManifest.RunAsUser()) {
+	// Even if the authorization policy allows it, apply another
+	// filter to keep apps from being deployed as system users.
```

Comment:
For future work - we should probably consolidate system users into the main authorization system since the question asked by the authz system specifically handles users and not apps. That has the nice side effect of making "systems" deployable with P2.